nx.json

```diff
@@ -73,7 +76,10 @@
         "{workspaceRoot}/.eslintrc.json",
         "{workspaceRoot}/.eslintignore",
         "{workspaceRoot}/eslint.config.js"
-      ]
+      ],
+      "options": {
+        "command": "eslint src"
```

Comment:
we override `lint` target default as nx eslint plugin infers target command to `eslint .` , but our current eslint setup relies on running it against `src/` otherwise it will explode.